google/cloud/bigtable/data/_async/client.py

```diff
@@ -131,12 +127,9 @@ def __init__(
                 on the client. API Endpoint should be set through client_options.
         Raises:
             RuntimeError: if called outside of an async context (no running event loop)
-            ValueError: if pool_size is less than 1
         """
-        # set up transport in registry
-        transport_str = f"pooled_grpc_asyncio_{pool_size}"
-        transport = PooledBigtableGrpcAsyncIOTransport.with_fixed_size(pool_size)
-        BigtableClientMeta._transport_registry[transport_str] = transport
+        if "pool_size" in kwargs:
```

Comment:
Could someone create a client with BigtableDataClientAsync("project-id", 3)?

Reply:
we discussed this offline, but the `__init__()` function includes a `*`, meaning only key-word arguments are allowed